### PR TITLE
QA tests & updates

### PR DESCRIPTION
**Pull request by drigodwin**

### CODE REVIEW

File: catalog.tests.bom [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+brooklyn.catalog:
+  items:
+  - id: rabbitmq-broker-tests
```
Comment (tbouron) [inline catalog.tests.bom:21]: Shouldn't be `brooklyn-rabbitmq-broker-tests` to match the catalog item?

**Comment by tbouron** [comment]
Minor comment.

Otherwise, tested, LGTM

File: catalog.tests.bom (hunk 1 above)
Comment (drigodwin) [inline catalog.tests.bom:21]: I've not been using a brooklyn prefix for any of the tests as I didn't think it was needed.

Comment (tbouron) [inline catalog.tests.bom:21]: No needed, just nice from a consistency point of view.

**Pull request merged**